planet/scripts/item_asset_types.py

```diff
@@ -31,8 +31,9 @@
 ]
 
 DEFAULT_ASSET_TYPES = [
-    'ortho_analytic_3b', 'ortho_analytic_4b', 'analytic_5b', 'analytic_5b_xml', 'ortho_analytic_8b',
-    'ortho_analytic_8b_xml','ortho_analytic_8b_sr', 'analytic_b1', 'analytic_b10', 'analytic_b11',
+    'ortho_analytic_3b', 'ortho_analytic_4b', 'analytic_5b',
```

Comment:
Only added the following to this list:
`analytic_8b_udm2`, `basic_analytic_8b_udm2`, `analytic_8b_sr_udm2`, `analytic_3b_udm2`

Large changes here were to fix issues with line length.